packages/realm/src/platform/node/custom-inspect.ts

```diff
@@ -0,0 +1,69 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+import { InspectOptionsStylized, inspect } from "node:util";
+
+import { Collection, Dictionary, List, RealmObject, RealmSet } from "../../internal";
+import { Results } from "realm/binding";
```

Comment:
Preferable to use imports via `"realm"` instead?

Reply:
Indeed - good catch, this is the wrong `Results`!